docs/installing.rst

```diff
@@ -55,8 +74,69 @@ Install behind a proxy::
   python get-pip.py --proxy="[user:passwd@]proxy.server:port"
 
 
-Upgrade pip
------------
+Installing with Linux Package Managers
+--------------------------------------
+
+Fedora
+~~~~~~
+
+To install the Fedora supported version of pip:
+
+* < Fedora 23:
+
+ * Python 2: ``sudo yum install python-pip``
+ * Python 3: ``sudo yum install python3-pip``
+
+* >= Fedora 23:
+
+ * Python 2: ``sudo dnf install python-pip``
+ * Python 3: ``sudo dnf install python3-pip``
```

Comment:
In F22+, python-pip and python3-pip should be installed by default with the corresponding Python runtime.